tests/foreman/ui/test_registration.py

```diff
@@ -0,0 +1,334 @@
+"""Tests for registration.
+
+:Requirement: Registration
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Registration
+
+:CaseAutomation: Automated
+
+:CaseImportance: Critical
+
+:Team: Rocket
+
+:TestType: Functional
+
+:Upstream: No
+"""
+import datetime
+
+import pytest
+from fauxfactory import gen_string
+
+from robottelo import constants
+from robottelo.config import settings
+from robottelo.constants import FAKE_1_CUSTOM_PACKAGE
+from robottelo.constants import FAKE_7_CUSTOM_PACKAGE
+
+
+@pytest.mark.no_containers
+@pytest.mark.tier3
+@pytest.mark.rhel_ver_match('[^6]')
+def test_positive_global_registration_end_to_end(
```

Comment:
seems like  `@pytest.mark.e2e` marker got lost while moving

Reply:
Missed the marker while moving. Updated.